### ISSUE TITLE
CK.1.1 sublineage with S:V1264L (111 seqs mainly in Japan, still circulating even shows growth advantage)

### ISSUE THREAD

**Issue by HynnSpylor** [cov-lineages/pango-designation#2271]
*Transferred from https://github.com/sars-cov-2-variants/lineage-proposals/issues/666
Defining mutations:
CK.1.1>C26907T>A27753C>G25352T (S:V1264L)
GISAID query: A24129G, G25352T, A27753C
Earliest seq: 2023-02-24 (Japan- EPI_ISL_17265415)
Most recent seq: 2023-08-18 (Japan- EPI_ISL_18122853)
Countries detected: Japan (108), South Korea (1), US (1)

Usher Tree:
![QQ截图20230907002601](https://github.com/cov-lineages/pango-designation/assets/121703496/cdf95783-8488-4d0e-8354-0d9a0f47c25f)
https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_3126b_8a6d50.json?c=gt-S_1264

In recent 3 months, almost (~98%) CK.1.1 (with SGTF and S:Y144-) seqs got S:V1264L. It seems very strange that an old BA.5* lineage can still circulate in Japan- Even it shows a 14% growth advantage:
![QQ截图20230907001316](https://github.com/cov-lineages/pango-designation/assets/121703496/b97b3930-0817-41d6-8b46-4f7e0a1eb96d)
https://cov-spectrum.org/explore/World/AllSamples/Past3M/variants?aaMutations=S%3AV1264L&nextcladePangoLineage=CK.1.1*&

And it becomes the biggest BA.5* lineages in past 3 months:
![QQ截图20230907002634](https://github.com/cov-lineages/pango-designation/assets/121703496/87945037-6ea9-45d1-9206-2c579477c364)

If we carefully analyze the Usher Tree, the ORF8:P38S may be another key mutations- the branch has increased and circulated since Jun 2023. 7 seqs further get S:L176F.

Despite it can not compete with XBB* outside Japan, it still deserves a further tracking.

Genomes:
EPI_ISL_17265415, EPI_ISL_17280829, EPI_ISL_17281014, EPI_ISL_17281546, EPI_ISL_17295219, EPI_ISL_17295221,
EPI_ISL_17387955, EPI_ISL_17388093, EPI_ISL_17388167, EPI_ISL_17463107, EPI_ISL_17495667, EPI_ISL_17495682,
EPI_ISL_17540021, EPI_ISL_17540023, EPI_ISL_17540025-17540027, EPI_ISL_17540176, EPI_ISL_17558606, EPI_ISL_17629485,
EPI_ISL_17630994, EPI_ISL_17631063, EPI_ISL_17631079, EPI_ISL_17637599-17637600, EPI_ISL_17652270, EPI_ISL_17880076,
EPI_ISL_17881491, EPI_ISL_17881691, EPI_ISL_17882706, EPI_ISL_17882776, EPI_ISL_17883773, EPI_ISL_17952860,
EPI_ISL_17973117, EPI_ISL_17973136, EPI_ISL_17980792, EPI_ISL_17982143, EPI_ISL_17985835, EPI_ISL_18006526,
EPI_ISL_18006568-18006569, EPI_ISL_18007023, EPI_ISL_18013331, EPI_ISL_18013347, EPI_ISL_18033119, EPI_ISL_18044112,
EPI_ISL_18051431, EPI_ISL_18051668, EPI_ISL_18051720, EPI_ISL_18056120, EPI_ISL_18060297, EPI_ISL_18060307,
EPI_ISL_18064876, EPI_ISL_18065198, EPI_ISL_18065265, EPI_ISL_18065270, EPI_ISL_18065290-18065291, EPI_ISL_18065909,
EPI_ISL_18066504, EPI_ISL_18067291, EPI_ISL_18067296, EPI_ISL_18067316, EPI_ISL_18071617, EPI_ISL_18082342,
EPI_ISL_18082733, EPI_ISL_18082764, EPI_ISL_18082768, EPI_ISL_18082773, EPI_ISL_18097036, EPI_ISL_18097142,
EPI_ISL_18097384, EPI_ISL_18097395, EPI_ISL_18097408, EPI_ISL_18097447, EPI_ISL_18106312, EPI_ISL_18106326-18106331,
EPI_ISL_18106335, EPI_ISL_18106337-18106339, EPI_ISL_18118051, EPI_ISL_18122670, EPI_ISL_18122733, EPI_ISL_18122817,
EPI_ISL_18122853, EPI_ISL_18122908, EPI_ISL_18122931, EPI_ISL_18122965, EPI_ISL_18122986, EPI_ISL_18127362,
EPI_ISL_18127575, EPI_ISL_18128140, EPI_ISL_18134206, EPI_ISL_18139646, EPI_ISL_18139662, EPI_ISL_18141048,
EPI_ISL_18146244, EPI_ISL_18164744, EPI_ISL_18164943, EPI_ISL_18214715, EPI_ISL_18214761, EPI_ISL_18215757,
EPI_ISL_18223426, EPI_ISL_18226568

**Comment by FedeGueli** [comment]
I think that the deletion S:Y144- (from CK.1.1) has its role in its success. We already saw that in BQ.1 when it had hit the wall fo the double BA.5/BA.5+346T recent immunity S:Y144- became widespread in the tree and Yunlong Cao confirmed with his  lab tests its importance. So my idea that the whole CK.1.1 managed to find in Japan its niche ( as we saw for other lineages in Australia, Philippines , New Zealand islands could permit that) and when BA.5 specific immunity started to wane it got the right mutation at residue 1264 that we already saw in "last warriors"(meaning more resilient lineages) as BA.2.38.3/BH.1 in India one of the last stepwise BA.2 to resist to saltations/recombinants in India.
